wormhole-connect/src/config/index.ts

```diff
@@ -69,15 +69,16 @@ export const USDC_BRIDGE_URL = config.cctpWarning?.href || '';
 export const WORMHOLE_RPC_HOSTS =
   ENV === 'MAINNET'
     ? [
-        'https://wormhole-v2-mainnet-api.certus.one',
-        'https://wormhole.inotel.ro',
         'https://wormhole-v2-mainnet-api.mcf.rocks',
         'https://wormhole-v2-mainnet-api.chainlayer.network',
         'https://wormhole-v2-mainnet-api.staking.fund',
-        'https://wormhole-v2-mainnet.01node.com',
```

Comment:
Can this use:
```
const MAINNET_GUARDIAN_RPC: string[] =  [
    "https://api.wormholescan.io", // Explorer offers a guardian equivalent endpoint for fetching VAAs
    "https://wormhole-v2-mainnet-api.mcf.rocks",
    "https://wormhole-v2-mainnet-api.chainlayer.network",
    "https://wormhole-v2-mainnet-api.staking.fund",
]
```

(from our docs https://docs.wormhole.com/wormhole/reference/sdk-docs)

Do we have to hit any Guardian RPC directly at all? On one hand depending on wormholescan is a bit centralized, but on the other, we already depend on wormholescan heavily for other status checks.

Reply:
We try to fetch the VAA from the Wormholescan API first and if that doesn't have the VAA, then we hit the guardians. If none of them have it then we wait 30 seconds and repeat this process.

Reply:
We do use the explorer api to fetch the VAA (see [here](https://github.com/wormhole-foundation/wormhole-connect/blob/development/wormhole-connect/src/utils/vaa.ts#L105)). The guardians are queried only if the vaa was not found on the api.